lib/jxl/jxl_test.cc

```diff
@@ -1475,7 +1475,7 @@ JXL_TRANSCODE_JPEG_TEST(JxlTest, RoundtripJpegRecompression444) {
   const std::vector<uint8_t> orig =
       ReadTestData("jxl/flower/flower.png.im_q85_444.jpg");
   // JPEG size is 696,659 bytes.
-  EXPECT_NEAR(RoundtripJpeg(orig, pool.get()), 568822u, 20);
+  EXPECT_NEAR(RoundtripJpeg(orig, pool.get()), 568879u, 20);
```

Comment:
Just a thought: 0.0035% threshold seems to be too tight. How about making it something like 0.05%

Reply:
yes, next time a test with such a small threshold fails, I'll fix it by bumping up the threshold instead...